test/escrow/curve/QuadraticCurveMath.t.sol

```diff
@@ -104,43 +94,57 @@ contract TestQuadraticIncreasingCurve is QuadraticCurveBase {
         // warmup complete
         vm.warp(block.timestamp + 1);
 
-        // python:              449.206279554928541696
-        // solmate (optimized): 449.206254284606635135
+        // excel:              420.6904013
         assertEq(
             curve.votingPowerAt(tokenIdFirst, block.timestamp),
-            449206254284606635135,
+            420690414675862320286,
             "Balance incorrect after warmup"
         );
         assertEq(curve.isWarm(tokenIdFirst), true, "Still warming up");
 
-        // python:    1067784543380942056100724736
-        // solmate:   1067784483312193385000000000
+        // excel:    1000000.954
         assertEq(
             curve.votingPowerAt(tokenIdSecond, block.timestamp),
-            1067784483312193385000000000,
+            1000000985704110676000000000,
             "Balance incorrect after warmup II"
         );
 
         // warp to the start of period 2
         vm.warp(start + clock.epochDuration());
-        // excel:     600.985714300000000000
-        // PRB:       600.985163959347100568
-        // solmate:   600.985163959347101852
-        // python :   600.985714285714341888
-        // solmate2:  600.985163959347101952
+        // excel:     436.8703846
+        assertEq(
+            curve.votingPowerAt(tokenIdFirst, block.timestamp),
+            436870384614923176704,
+            "Balance incorrect after p1"
+        );
+
+        // check at 6 months
+        vm.warp(start + 26 weeks);
+        // excel:     631.035
+        assertEq(
+            curve.votingPowerAt(tokenIdFirst, block.timestamp),
+            631034999994001297152,
+            "Balance incorrect after p1"
+        );
+
+        // check at 2 years
+        vm.warp(start + 104 weeks);
+        // excel:     1262.07
         assertEq(
             curve.votingPowerAt(tokenIdFirst, block.timestamp),
-            600985163959347101952,
+            1262069999976005188608,
             "Balance incorrect after p1"
         );
 
-        uint256 expectedMaxI = 2524126241845405205760;
-        uint256 expectedMaxII = 5999967296216704000000000000;
+        // 1682.76
+        uint256 expectedMaxI = 1682759999964007782912;
+        // 4000000
+        uint256 expectedMaxII = 3999999999914444800000000000;
```

Comment:
Just double checking. What is the max expected VP?

Reply:
4x